TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -226,6 +226,8 @@ def analyze(self,event):
     nrunning = 0
     if options.queue == "condor":
         running = subprocess.check_output(["condor_q", "-nobatch","-wide"])
+    elif options.queue == "cp3":
+        running = subprocess.check_output(["squeue", "-u", "vischia"])
```

Comment:
could this be os.environ["USER"] instead of "vischia", so we can also use this with our slurm setup in oviedo? Thanks

Reply:
Done, will commit once the comment by Giovanni is resolved